### ISSUE TITLE
Eclipse New Connection panel: can tab into non-editable areas

### ISSUE THREAD

**Issue by mattcolegate** [eclipse/codewind#776]
**Codewind version:** latest hybrid
**OS:** Windows 10

**Che version:** N/A
**IDE extension version:** 0.5.0v201910171920
**IDE version:** 2019-06
**Kubernetes cluster:** N/A

**Description:** Tabbing through the panel tabs into the labels 'Create a connection to a Codewind instance' and 'Fill in the deployment information:'. 

**Steps to reproduce:** Right-click Codewind in the Codewind Explorer view and select New Connection. Press the tab key to cycle through the fields in the resulting panel. As well as editable fields and the buttons, the above labels also can be selected.

**Workaround:** N/A

**Comment by eharris369** [comment]
This is intentional for accessibility purposes - some screen readers do not read labels on lines by themselves so we use read only text instead.  I did find a way to read labels with MAC voice over.  I could check if JAWS has a way as well.  I don't know what other screen readers we target.

**Comment by eharris369** [comment]
This affects the New Connection wizard and Add Template Source page of the Manage Templates sources dialog.

**Comment by eharris369** [comment]
I found a way in JAWS to read the labels by using Inspect (JAWS key) + B.  This reads the entire dialog from start to finish including the labels.  Since labels are usually just extra information, this may be ok.  Sent an email to an expert to check - just waiting for a response.  I did remove one read only text and replaced it with a better group label (which JAWS does read).  This will not work for longer descriptions.
